config.json

```diff
@@ -533,10 +539,10 @@
             "BuildNumberMinor": "default"
           }
         },
-        "portableLinux":{
+        "portable":{
           "description": "Download packages that are portable over glibc based Linux distros.",
           "settings": {
-            "RuntimeOS":"linux"
+            "PortableBuild":"1"
```

Comment:
For consistency with other msbuild properties can we make this "true" instead of "1"?